libmultilabel/networks/bigru.py

```diff
@@ -0,0 +1,60 @@
+from math import floor
+
+import torch
+import torch.nn as nn
+from torch.nn.init import xavier_uniform_
+from torch.nn.utils.rnn import pack_padded_sequence, pad_packed_sequence
+
+from ..networks.base import BaseModel
+
+
+class BiGRU(BaseModel):
+    def __init__(
+        self,
+        embed_vecs,
+        num_classes,
+        rnn_dim=512,
+        num_layers=1,
```

Comment:
Now I feel that rnn_layers may be better. num_layers sounds like the number of layers of the whole network (including attention and linear etc.)